components/automate-ui/src/app/services/projects-filter/projects-filter.effects.ts

```diff
@@ -30,13 +30,13 @@ export class ProjectsFilterEffects {
 
   private POLLING_INTERVAL_IN_SECONDS = 120; // 2 minutes
 
-  @Effect()
-  latestOptions$ = observableInterval(1000 * this.POLLING_INTERVAL_IN_SECONDS).pipe(
-    mergeMap(this.loadOptionsAction$()));
+  latestOptions$ = createEffect(() =>
+    observableInterval(1000 * this.POLLING_INTERVAL_IN_SECONDS).pipe(
+    mergeMap(this.loadOptionsAction$())));
 
   // Fast-initialize project filter just from local storage
-  @Effect()
-  initOptions$ = this.actions$.pipe(
+  initOptions$ = createEffect(() => {
+    return this.actions$.pipe(
```

Comment:
Let's be consistent: as you have it here, there are two equivalent syntaxes for writing the `createEffect` expression. I want to urge the more concise format (L33). No need to introduce a separate block of code (L38-39) when there is just a single return. (Apologies that that does make a lot of editing for you. 😁  )